fdbserver/FDBExecHelper.actor.cpp

```diff
@@ -194,14 +199,15 @@ ACTOR Future<int> spawnProcess(std::string path,
 			} else {
 				// child process completed
 				if (!(WIFEXITED(status) && WEXITSTATUS(status) == 0)) {
-					TraceEvent event(SevWarnAlways, "SpawnProcess : Command failed");
+					TraceEvent event(SevWarnAlways, "SpawnProcessFailure");
 					setupTraceWithOutput(event, bytesRead, outputBuffer);
-					event.detail("Cmd", path)
+					event.detail("Reason", "Command failed")
+					    .detail("Cmd", path)
 					    .detail("Args", allArgs)
 					    .detail("Errno", WIFEXITED(status) ? WEXITSTATUS(status) : -1);
 					return WIFEXITED(status) ? WEXITSTATUS(status) : -1;
 				}
-				TraceEvent event("SpawnProcess : Command status");
+				TraceEvent event("SpawnProcess_CommandStatus");
```

Comment:
It's more typical to omit the underscore, though there are examples either way.

```suggestion
				TraceEvent event("SpawnProcessCommandStatus");
```